rackspace/compute/v2/servers/results.go

```diff
@@ -0,0 +1,179 @@
+package servers
+
+import (
+	"reflect"
+
+	"github.com/mitchellh/mapstructure"
+	"github.com/rackspace/gophercloud"
+	"github.com/rackspace/gophercloud/pagination"
+)
+
+// Server represents a Rackspace server on the user's account.
+type Server struct {
+	// ID uniquely identifies this server amongst all other servers, including those not accessible to the current tenant.
+	ID string
```

Comment:
We _could_, but I don't think we gain anything here by doing it. Specifically, if the OpenStack server has fields that Rackspace doesn't have (now or in the future), we'll end up copy-and-pasting like this anyhow. Only if done that way, it would potentially break code that initializes a Rackspace server (since it'll no longer embed an OpenStack server):
```
server := rsServers.Server{
   osServers.Server{

   },
}
```